README.md

```diff
@@ -80,4 +84,4 @@ If you have any issues with this library or encounter any bugs then please get i
 
 ---
 
-License: MIT
+License: [MIT](https://opensource.org/licenses/MIT)
```

Comment:
Can we make the link point to LICENSE.txt instead?

Reply:
yeah, good idea

Reply:
done